happytransformer/adaptors/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .adaptor import Adaptor
+from .albert import AlbertAdaptor
+from .berts import BertAdaptor,DistilBertAdaptor,RobertaAdaptor
+
+ADAPTORS = {
+    'BERT':BertAdaptor(),
+    'DISTILBERT':DistilBertAdaptor(),
+    'ROBERTA':RobertaAdaptor(),
+    'ALBERT':AlbertAdaptor()
+}
+
+def get_adaptor(model_type:str)->Adaptor:
+    if model_type in ADAPTORS:
```

Comment:
Only BERT can perform NSP. The user would eventually come across a "NotImplementedError()" but it's better if we give a ValueError with a proper output.

Reply:
Good point. I can update the default error in `Adaptor` to say that X type of transformer model doesn't support Y feature